packages/rocketchat-livechat/client/views/app/livechatAppearance.html

```diff
@@ -66,6 +66,13 @@ <h2>{{_ "Settings"}}</h2>
 						<textarea class="preview-settings rc-input__element" name="offlineSuccessMessage" id="offlineSuccessMessage">{{offlineSuccessMessage}}</textarea>
 					</div>
 				</fieldset>
+				<fieldset>
+					<legend>{{_ "Conversation_finished"}}</legend>
```

Comment:
`i18n` string is missing

Reply:
I guess we already have this `i18n` strings: [https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-livechat/.app/i18n/pt-BR.i18n.json#L11](url)

Reply:
🤦‍♂️  my global search excludes i18n JSON files. sry 😬